### PR TITLE
feat(app): render deckview in labware setup and module setup steps

### PR DESCRIPTION
**Pull request by shlokamin**
# Overview
This PR renders the OT-3 deckview in the module setup + labware setup steps in the set up for run flow

closes RCORE-108

# Changelog

- Render OT-3 deckview in protocol setup, labware setup, and liquid setup
- Update robot schema to reflect robotType rather than robotName

# Risk assessment

Low/med

### CODE REVIEW

File: app/src/organisms/Devices/ProtocolRun/SetupLiquids/SetupLiquidsMap.tsx [hunk 1]
```diff
@@ -75,9 +81,9 @@ export function SetupLiquidsMap(props: SetupLiquidsMapProps): JSX.Element {
       justifyContent={JUSTIFY_CENTER}
     >
       <RobotWorkSpace
-        deckDef={(standardDeckDef as unknown) as DeckDefinition}
-        viewBox={DECK_MAP_VIEWBOX}
-        deckLayerBlocklist={DECK_LAYER_BLOCKLIST}
+        deckDef={deckDef}
+        viewBox={getViewBox(robotType)}
```
Comment (brenthagen) [inline app/src/organisms/Devices/ProtocolRun/SetupLiquids/SetupLiquidsMap.tsx:83]: can this use `getStandardDeckViewBox` also?

**Review by brenthagen — APPROVED** [review]
once conflicts/js-checks resolved, LGTM !

Screen Shot 2022-11-02 at 2 58 32 PM

File: app/src/organisms/Devices/hooks/useLabwareRenderInfoForRunById.ts [hunk 2]
```diff
@@ -1,20 +1,19 @@
-import standardDeckDef from '@opentrons/shared-data/deck/definitions/3/ot2_standard.json'
-
 import { getLabwareRenderInfo } from '../ProtocolRun/utils/getLabwareRenderInfo'
 import { useProtocolDetailsForRun, useStoredProtocolAnalysis } from '.'
 
 import type { LabwareRenderInfoById } from '../ProtocolRun/utils/getLabwareRenderInfo'
+import { getDeckDefFromRobotType } from '@opentrons/shared-data'
```
Comment (koji) [inline app/src/organisms/Devices/hooks/useLabwareRenderInfoForRunById.ts]: should be the first line?

**Review by koji — APPROVED** [review]
🚀

File: app/src/organisms/Devices/hooks/useLabwareRenderInfoForRunById.ts (hunk 2 above)
Comment (shlokamin) [reply app/src/organisms/Devices/hooks/useLabwareRenderInfoForRunById.ts]: yes, thank you!

File: app/src/organisms/Devices/ProtocolRun/SetupLiquids/SetupLiquidsMap.tsx (hunk 1 above)
Comment (shlokamin) [reply app/src/organisms/Devices/ProtocolRun/SetupLiquids/SetupLiquidsMap.tsx:83]: unfortunately not, the viewbox is slightly different in liquid setup :(

**Pull request merged**